src/aleph/schemas/validated_message.py

```diff
@@ -29,6 +28,14 @@
 from .message_content import MessageContent
 
 
+class MessageConfirmation(BaseModel):
+    chain: Chain
+    height: int
+    hash: str
+    time: float
```

Comment:
What about using `pydantic.Field` to document and describe these fields in the model ?